examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -44,6 +48,37 @@ class Task extends TypedEventEmitter<ITaskEvents> implements ITask {
             this.emit("priorityChanged");
         });
     }
+    private readonly resetDiffProperties = (): void => {
+        this.diffPriority = DEFAULT_DIFF_PRIORITY;
+        this.diffType = "none";
+        this.emit('externalPriorityChanged');
+        this.diffName = '';
+        this.diffType = "none";
+        this.emit('externalNameChanged');
+    }
+    public externalNameChanged = (savedName: string): void => {
+        this.diffName = savedName;
+        this.diffType = "change";
+        this.emit('externalNameChanged');
+    }
+    public externalPriorityChanged = (savedPriority: number): void => {
+        this.diffPriority = savedPriority;
+        this.diffType = "change";
+        this.emit('externalPriorityChanged');
+    }
+    public acceptChange = (): void => {
+        if (this.diffPriority !== DEFAULT_DIFF_PRIORITY) {
+            this._priority.set(this.diffPriority);
+        }
+        if (this.diffName !== '') {
+            const oldString = this._name.getText()
+            this._name.replaceText(0, oldString.length, this.diffName);
+        }
+        this.resetDiffProperties();
+    }
+    public ignoreChange = (): void => {
```

Comment:
I think ignore probably shouldn't be an option - i.e. it's ok to delay accepting, but ultimately the goal is to converge.  Maybe eventually this becomes something like saveToSource (try to write the fluid data back to the source) but not needed for now.